packages/cli/src/modules/chat-hub/chat-hub.service.ts

```diff
@@ -402,6 +403,7 @@ export class ChatHubService {
 				message,
 				payload.credentials,
 				payload.model,
+				session.lastMessageAt === null, // generate title on receiving the first human message only
```

Comment:
Would `payload.previousMessageId` being null be a better condition? It would be null on the edit of the first message too, but updating the title in that case doesn't seem that wrong..